### PR TITLE
hotfix(themes: fix price option with discount

### PR DESCRIPTION
**Pull request by Ahmed-Basalib10**
What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

* Bug

What is the current behaviour? (You can also link to an open issue here)

* Price not updated when the option has discount

What is the new behaviour? (You can also link to the ticket here)

* Price updated according to the option discount

Does this PR introduce a breaking change?

* No

Screenshots (If appropriate)

*

### CODE REVIEW

File: src/views/pages/product/single.twig [hunk 1]
```diff
@@ -80,6 +80,7 @@
 | product.add_to_cart_label              | string                      |                                                                        |
 | product.currency                       | string                      |                                                                        |
 #}
+{% set product_is_on_sale = product.is_on_sale and product.regular_price > product.sale_price %}
```
Comment (jalmatari) [inline src/views/pages/product/single.twig:83]: no need for this, use it as is, `product.is_on_sale` if there is edge case related to it, should be done from back end

**Review by jalmatari — CHANGES_REQUESTED** [review]

**Review by jalmatari — APPROVED** [review]

**Comment by SallaDev** [comment]
**Status Checks:**
- Please make sure  `Codacy Static Code Analysis` of (https://github.com/SallaApp/theme-raed/pull/317) is passed  ❌.

Merge by: @jalmatari

**Comment by SallaDev** [comment]
https://github.com/SallaApp/theme-raed/pull/317 is been merged ✅

-------------------

Merge by: @jalmatari

**Pull request merged**